lib/plugins/aws/deploy/index.test.js

```diff
@@ -189,6 +193,12 @@ describe('AwsDeploy', () => {
       })
     );
 
+    it('should run "aws:deploy:deploy:validateTemplate" hook', () => awsDeploy
+      .hooks['aws:deploy:deploy:validateTemplate']().then(() => {
+        expect(validateTemplateStub.calledOnce).to.equal(true);
```

Comment:
if you use `chai-as-promised` and `sinon-chai` (with chai.use() ) this can be written as:

`expect(validateTemplateStub).to.have.been.calledOnce`

There are multiple locations where it can be changed.